crates/subspace-networking/Cargo.toml

```diff
@@ -47,9 +47,8 @@ tracing-subscriber = { version = "0.3.16", features = ["env-filter"]}
 unsigned-varint = { version = "0.7.1", features = ["futures", "asynchronous_codec"] }
 
 [dependencies.libp2p]
-# TODO: change to upstream release when https://github.com/libp2p/rust-libp2p/pull/3287 is released
-git = "https://github.com/subspace/rust-libp2p"
-rev = "917b388b0549810903946664a61c9b313b2e9fad"
+git = "https://github.com/libp2p/rust-libp2p"
+rev = "3c5940aeadb9ed8527b6f7aa158797359085293d"
```

Comment:
We can set the crates' version but we often switch to a specific revision. Please, let me know if you'd like to see crates' version instead.

Reply:
Yes, version from crates.io is preferred, it was never the goal to use git commit if we can avoid it.

Reply:
Even if you use a version from `crates.io`, crate's hash is written to `Cargo.lock` so you can also be sure that you can reproduce your build. It is just that update versions is much easier with `crates.io` (just run `cargo update` and it will update hashes and revisions automatically)